### ISSUE TITLE
Accidentally deleted my publisher

### ISSUE THREAD

**Issue by stoneC0der** [microsoft/vscode-vsce#547]
I was having issue when trying to upload my extension, the error was the JSON output below

`
{"$id":"1","customProperties":{"Descriptor":null,"IdentityDisplayName":null,"Token":null,"RequestedPermissions":0,"NamespaceId":"00000000-0000-0000-0000-000000000000"},"innerException":null,"message":"Access Denied: The Personal Access Token used has expired.","typeName":"Microsoft.VisualStudio.Services.Security.AccessCheckException, Microsoft.VisualStudio.Services.WebApi","typeKey":"AccessCheckException","errorCode":0,"eventId":3000}
`

After some googling I decided to look at vsce help from the terminal to found additional command I was not familiar with, one of these was to list all publisher, logout and delete

I mistakenly created several publishers but the issue was that the original theme was StoneC0der but I at some point created a PAT using the wrong spelling instead of StoneC0der I had Stonec0der & stoneCoder.
Since I couldn't not figure out the correct publisher ID (which I did later on but too).

I decided to logout all publishers and then was tempted to delete them since I was now getting error like this:

`Error: Access Denied: <sometoken was here> needs the following permission(s) on the resource /StoneC0der to perform this action: View user permissions on a resource`

Unfortunately (did not expected it but feel natural) my extension was gone from the market place.
I created a publisher with the same ID and had to republish the extension

My question is, is everything really lost?
I notice that after publishing the extension et searching for it from vscode show me the install button and all the metrics are gone.

Basically it seams that those who using it will not see this update, site like vscodethemes still show the theme with their own metric but point to the newly published theme.

I guess what I want to know if it possible to get the metrics back (really don't think it possible but just hope) and how to notify users about this unexpected change.

Thanks and sorry I sound confusing

**Comment by joaomoreno** [comment]
Moved to https://github.com/microsoft/vsmarketplace/issues/32
